sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -279,11 +273,16 @@ object Webpack {
     * @param workingDir Working directory in which the Nodejs will be run (where there is the `node_modules` subdirectory)
     * @param log Logger
     */
-  def run(nodeArgs: String*)(args: String*)(workingDir: File, log: Logger): Option[WebpackStats] = {
+  def run(nodeArgs: String*)(args: String*)(workingDir: File, log: Logger): WebpackStats = {
     val webpackBin = workingDir / "node_modules" / "webpack" / "bin" / "webpack"
     val params = nodeArgs ++ Seq(webpackBin.absolutePath, "--profile", "--json") ++ args
     val cmd = "node" +: params
-    Commands.run(cmd, workingDir, log, jsonOutput(cmd, log)).fold(sys.error, _.flatten)
+    log.debug(s"Running command [${cmd.mkString(" ")}]")
+    Commands.run(cmd, workingDir, log, jsonOutput(cmd, log))
+      .fold(
+        sys.error,
```

Comment:
If you're going to throw when the `Try` is a `Failure`, you as well do
```scala
val runResult = Commands.run(...).get
```
But then again, why change `Commands.run` to return a `Try` in that case, instead of directly throwing the exception?